android-components/components/feature/customtabs/src/test/java/mozilla/components/feature/customtabs/CustomTabsToolbarFeatureTest.kt

```diff
@@ -441,6 +441,84 @@ class CustomTabsToolbarFeatureTest {
         assertEquals(24, button.drawable.intrinsicWidth)
     }
 
+    @Test
+    fun `wide action button drawable is scaled to 48 width and 24 height`() {
+        val captor = argumentCaptor<Toolbar.ActionButton>()
+        val width = 96
+        val height = 48
+        val pendingIntent: PendingIntent = mock()
+        val tab = createCustomTab(
+            "https://www.mozilla.org",
+            id = "mozilla",
+            config = CustomTabConfig(
+                actionButtonConfig = CustomTabActionButtonConfig(
+                    description = "Button",
+                    icon = Bitmap.createBitmap(IntArray(width * height), width, height, Bitmap.Config.ARGB_8888),
+                    pendingIntent = pendingIntent,
+                ),
+            ),
+        )
+        val store = BrowserStore(
+            BrowserState(
+                customTabs = listOf(tab),
+            ),
+        )
+        val toolbar = spy(BrowserToolbar(testContext))
+        val useCases = CustomTabsUseCases(
+            store = store,
+            loadUrlUseCase = SessionUseCases(store).loadUrl,
+        )
+        val feature = spy(CustomTabsToolbarFeature(store, toolbar, sessionId = "mozilla", useCases = useCases) {})
+
+        feature.start()
+
+        verify(feature).addActionButton(anyInt(), any())
+        verify(toolbar).addBrowserAction(captor.capture())
+
+        val button = captor.value.createView(FrameLayout(testContext))
+        assertEquals(24, (button as ImageButton).drawable.intrinsicHeight)
+        assertEquals(48, button.drawable.intrinsicWidth)
+    }
+
+    @Test
+    fun `tall action button drawable is scaled to 12 width and 24 height`() {
```

Comment:
let's get this in the "given, when, then" format. Maybe something like:

`GIVEN the action button is tall WHEN it's being drawn in the toolbar THEN scale it to 12 width and 24 height`